salt/modules/cmdmod.py

```diff
@@ -245,6 +245,10 @@ def _run(cmd,
     '''
     Do the DRY thing and only call subprocess.Popen() once
     '''
+    # Remove whitespace from strings
+    if isinstance(cmd, six.string_types):
+        cmd = cmd.strip()
+
```

Comment:
@terminalmage I did follow it, but it didn't actually take me to a specific line number so I wasn't sure what you were talking about. I assume you meant the the shlex.split... If I recall correctly, Mike had me add that for Shell Injection checking or something...

Anyways, yours works too. And you don't need the `cmd.strip()` in the call to `win_runas`. It must have been the `shlex.split()` that was causing the problem... So, if you don't need the shlex for shell injection checking, then by all means merge yours.